migrations/20200427161609-add-body-and-description-to-position.js

```diff
@@ -1,8 +1,8 @@
 module.exports = {
-  up: (queryInterface, Sequelize) => queryInterface.addColumn(
-      'positions',
-      'body_id',
-      { type: Sequelize.INTEGER, allowNull: false }
-  ),
-  down: queryInterface => queryInterface.removeColumn('postiions', 'body')
+    up: (queryInterface, Sequelize) => queryInterface.addColumn(
+        'positions',
+        'body_id',
+        { type: Sequelize.INTEGER, allowNull: false }
```

Comment:
also here it should be nullable

Reply:
haha yeah just figures it out from the failed tests 😓